torch/csrc/jit/frontend/ir_emitter.h

```diff
@@ -13,7 +13,7 @@
 namespace torch {
 namespace jit {
 
-TORCH_API void runCleanupPasses(std::shared_ptr<Graph>& to_clean);
+TORCH_API void RunCleanupPasses(std::shared_ptr<Graph>& to_clean);
```

Comment:
Why Capitalizing™ this?

Reply:
The other passes (e.g. `ConvertToSSA`, `CanonicalizeModifiedLoops`, `NormalizeOps`) are capitalized, so I thought we should maintain that consistency. I can switch it back, though

Reply:
I see! One small issue is that it's hard to tell whether we are running a plain function or constructing an object in place by looking at `PassName(...)` call, since typically we capitalize type names, although it's easier to tell it's a function call here. I guess this is not urgent :), but it'd be great if we can rename all passes to start with lower case char.

Reply:
I like that better; I'll rename all the passes to be in `camelCase` if they aren't objects